components/scream/src/physics/shoc/tests/shoc_energy_update_dse_tests.cpp

```diff
@@ -38,8 +38,8 @@ struct UnitWrap::UnitTest<D>::TestShocUpdateDse {
 
     // Liquid water potential temperature [K]
     static constexpr Real thlm[nlev] = {350, 325, 315, 310, 300};
-    // Exner function [-]
-    static constexpr Real exner[nlev] = {0.1, 0.3, 0.5, 0.7, 1.0};
+    // Inverse of the Exner function [-]
+    static constexpr Real inv_exner[nlev] = {0.1, 0.3, 0.5, 0.7, 1.0};
```

Comment:
@bogensch Do these values actually make sense for `inv_exner`? If I change to the inverse of these values this test (`shoc_energy_host_dse_property`) will fail with `REQUIRE(host_dse[k+1] - host_dse[k] < 0)`.

Reply:
@tcclevenger yes, this is wrong. BUT, I already fixed this for PR #1028 , when I pushed the change into @AaronDonahue branch (https://github.com/E3SM-Project/scream/commit/8cf79bbfb256e9c411bd73d58ac5a48597fa86f0https://github.com/E3SM-Project/scream/commit/8cf79bbfb256e9c411bd73d58ac5a48597fa86f0https://github.com/E3SM-Project/scream/commit/8cf79bbfb256e9c411bd73d58ac5a48597fa86f0https://github.com/E3SM-Project/scream/commit/8cf79bbfb256e9c411bd73d58ac5a48597fa86f0).  However, for some reason this did not make it onto master?  I see on May 7th on PR #1028 @AaronDonahue did a force push of this branch so may be something went wrong there?  In any case, this should be fixed.  @tcclevenger should do you want me to modify/fix these and push to your branch?

Reply:
Ahhhhi imagine this was a case where we had to rebase off of a new master. Which requires a force push. Somehow the last commit didn't make it in. My apologies for that error

Reply:
@bogensch Yes, that would be good if you could just push the changes here.

Reply:
Not sure if the fix is in yet, but we can just cherry-pick the git hash that Peter B posted.

Reply:
@AaronDonahue or you can just admit this is part of your sneaky plan to sabotage my career ;) .  Kidding, I should have have noticed myself when I signed off on PR #1028 .  @tcclevenger I pushed the corrected tests.